elements/lisk-tree/src/sparse_merkle_tree/utils.ts

```diff
@@ -182,8 +184,31 @@ export const binaryStringToBuffer = (str: string) => {
 	return buf;
 };
 
-export const leafHash = (key: Buffer, value: Buffer): Buffer =>
-	hash(Buffer.concat([LEAF_HASH_PREFIX, key, value]));
+export const getLeafData = (data: Buffer, keyLength: number): { key: Buffer; value: Buffer } => {
```

Comment:
maybe rename for better readability
```suggestion
export const parseLeaf = (data: Buffer, keyLength: number): { key: Buffer; value: Buffer } => {
```